strands_human_aware_navigation/scripts/human_aware_navigation.py

```diff
@@ -116,7 +110,8 @@ def resetSpeed(self):
 
     def pedestrianCallback(self, pl):
         if not self._as.is_active():
-            rospy.logdebug("No active goal")
+            rospy.logdebug("No active goal. Unsubscribing.")
+            self.ppl_sub = None
```

Comment:
Is this enough? Does one not call [`Subscriber.unregister()`](http://docs.ros.org/indigo/api/rospy/html/rospy.topics.Subscriber-class.html#unregister)? I really don't know, just wondered...